server/base/src/test/java/org/apache/accumulo/server/fs/VolumeManagerImplTest.java

```diff
@@ -104,4 +111,88 @@ public Scope getChooserScope() {
       assertThrows(RuntimeException.class, () -> vm.choose(chooserEnv, volumes));
     }
   }
+
+  @Test
+  public void testConfigurationOverrides() throws Exception {
+
+    final String vol1 = "file://127.0.0.1/vol1/";
+    final String vol2 = "file://localhost/vol2/";
+    final String vol3 = "hdfs://127.0.0.1/accumulo";
+    final String vol4 = "hdfs://localhost/accumulo";
+
+    ConfigurationCopy conf = new ConfigurationCopy();
+    conf.set(Property.INSTANCE_VOLUMES, String.join(",", vol1, vol2, vol3, vol4));
+    conf.set(Property.GENERAL_VOLUME_CHOOSER, Property.GENERAL_VOLUME_CHOOSER.getDefaultValue());
+    conf.set(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1 + "."
+        + HdfsClientConfigKeys.HedgedRead.THREADPOOL_SIZE_KEY, "10");
+    conf.set(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1 + "."
```

Comment:
Could set a third override on one volume that no others set to ensure there is no interference for this case.  For example could set a third blocksize prop for only vol1.

Reply:
Added blocksize property to one volume in test.